tools/slang-test/slang-test-main.cpp

```diff
@@ -1100,6 +1103,21 @@ static TestResult runExecuteC(TestContext* context, TestInput& input)
     auto filePath = input.filePath;
     auto outputStem = input.outputStem;
 
+    // Make the module name the same as the source file
+    String directory = Path::getParentDirectory(input.outputStem);
+    String moduleName = Path::getFileNameWithoutExt(filePath);
+
+    String modulePath = Path::combine(directory, moduleName);
+
+    CPPCompileOptions options;
+
+    // Compile this source
+    options.sourceFiles.add(filePath);
+    options.modulePath = modulePath;
+
+    ExecuteResult exeRes;
+
+#ifdef _WIN32
```

Comment:
It seems like `WinVisualStudioUtil` and `UnixCPPCompilerUtil` have enough shared API that we could hide some of the platform-specific-ness here behind a common `CPPCompilerUtil` API.